src/components/il-image-feature/il-image-feature.component.js

```diff
@@ -11,16 +11,22 @@ class ImageFeatureComponent extends LitElement {
   
   static get properties() {
     return {
-      compact: { default: false, attribute: false }
+      _compact: { default: false, type: Boolean, attribute: false, state: true },
+      linkedby: { default: false, type: String, attribute: true },
+      highlight: {default: false, type: Boolean, attribute: true, reflect: true}
     };
   }
 
   static hasContainerSupport() {
     return CSS.supports('container-type', 'inline-size');
   }
 
+  static determineClickable() {
+    return CSS.supports('container-type', 'inline-size');
+  }
+
   handleResize(evt) {
-    this.compact = this.offsetWidth < 768;
+    this._compact = this.offsetWidth < 768;
```

Comment:
I don't like burying this number so deep in the code. At minimum we should make this a constant and have it closer to the top of the file